common/source/docs/common-frsky-repurposed.rst

```diff
@@ -63,33 +97,7 @@ When configured to display “GPS,” the custom telemetry screen will show long
 .. image:: ../../../images/OpenTX_GPSdisplay.png
  :target: ../_images/OpenTX_GPSdisplay.png
 
-Protocol information
---------------------
-
-Values that are sent over the FrSky telemetry link by ArduPilot:
-
-+-------------------------------------+---------------------------------------------------------------+
-| Taranis telemetry screen identifier |                          Description                          |
-+=====================================+===============================================================+
-| Fuel                                | Remaining battery capacity %                                  |
-+-------------------------------------+---------------------------------------------------------------+
-| VFAS                                | Battery voltage                                               |
-+-------------------------------------+---------------------------------------------------------------+
-| Curr                                | Current consumption                                           |
-+-------------------------------------+---------------------------------------------------------------+
-| GPS                                 | Latitude/longitude                                            |
-+-------------------------------------+---------------------------------------------------------------+
-| GSpd                                | GPS groundspeed                                               |
-+-------------------------------------+---------------------------------------------------------------+
-| GAlt                                | GPS altitude                                                  |
-+-------------------------------------+---------------------------------------------------------------+
-| Hdg                                 | Yaw angle                                                     |
-+-------------------------------------+---------------------------------------------------------------+
-| Alt                                 | Navigation altitude (relative to home)                        |
-+-------------------------------------+---------------------------------------------------------------+
-| Tmp1                                | Control/flight mode                                           |
-+-------------------------------------+---------------------------------------------------------------+
-| Tmp2                                | GPS status and number of satellites (as num_sats*10 + status) |
-+-------------------------------------+---------------------------------------------------------------+
+Configuringation with ErSky9x
+=============================
 
-If you installed FrSky sensors on your vehicle, other messages from these may also appear during discovery.
\ No newline at end of file
+For information on how to configure your ErSky9x transmitter (such as a Turnigy 9XR Pro) for FrSky telemetry, please go :ref:`here <common-frsky-telemetry>`.
```

Comment:
Thanks for reading.  That's not a typo. "ErSky9x" is the name of the firmware with telemetry support. You can read more about it, e.g., http://openrcforums.com/forum/.